KarateTraining/src/test/resources/signInSignUpResponse.json

```diff
@@ -0,0 +1,10 @@
+{
+  "user":{
+    "id":"#number",
+    "email":"#string",
+    "username":"#string",
+    "bio":null,
+    "image":null,
```

Comment:
are those fields always null?

Reply:
Yes, but now that i know how to use "##", i can use it here to check if is null or if recieve a string!